src/main/java/com/ibm/g11n/pipeline/client/DocumentType.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright IBM Corp. 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ibm.g11n.pipeline.client;
+
+/**
+ * Document type enum.
+ *
+ * @author John Emmons
+ */
+public enum DocumentType {
```

Comment:
It might be obvious, but please provide API doc for enum and each entry.

Reply:
Yes, it is obvious, but OK no problem.